lib/rouge/lexers/erlang.rb

```diff
@@ -77,7 +77,7 @@ class Erlang < RegexLexer
         rule %r{(#{atom_re})(:)} do
           groups Name::Namespace, Punctuation
         end
-        rule %r{(?:^|(?<=:))(#{atom_re})(\s*)(\()} do
+        rule %r{(#{atom_re})(\s*)(\()} do
```

Comment:
I have no idea what these were for, they don't seem to affect any actual tokens.